### PR TITLE
Minor changes to ppp format, update quickstart section of UG

### PR DESCRIPTION
**Pull request by rebeccacxy**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -18,8 +18,7 @@ Fridgy is a **food and recipe management system**, for users to manage their ing
 Fridgy helps you track your ingredients and recipes. It also warns you about expiring and expired ingredients, and automatically deducts your ingredients when you cook recipes.
 
 Fridgy is made for people living in shared spaces, since fridges can get *very messy* and difficult to keep track of!
-
-Fridgy is a *desktop app*, optimized for use via a Command Line Interface (CLI), while having an interactive Graphical User Interface (GUI) to display ingredients and recipes! If you can type fast, Fridgy can get your fridge Inventory and recipe sorted out faster than traditional GUI apps.
+It is a *desktop app*, optimized for use via a Command Line Interface (CLI), with an interactive Graphical User Interface (GUI) to display ingredients and recipes! If you can type fast, Fridgy can get your fridge Inventory and recipe sorted out faster than traditional GUI apps.
```
Comment (NayLin-H99) [inline docs/UserGuide.md:21]:
```suggestion
It is a *desktop app*, optimized for use via a Command Line Interface (CLI), with an interactive Graphical User Interface (GUI) to display ingredients and recipes! If you can type fast, Fridgy can get your fridge Inventory and Recipe Book sorted out faster than traditional GUI apps.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -57,26 +56,29 @@ This User Guide covers:
 
 Now that Fridgy is installed, you can test out the commands — add ingredients, recipes, and more!
 
-Adding our first ingredient:
+Let us add our first ingredient:
 
 `add ingredient -n Grapes -q 100g -e 25-10-2022 -d Seedless grapes`
 
+This adds an ingredient named Grapes, with a quantity of 100g, expiry date of 25-10-2022, and description of "Seedless grapes".
```
Comment (NayLin-H99) [inline docs/UserGuide.md:63]:
```suggestion
This adds an ingredient named "Grapes", with a quantity of 100g, an expiry date of 25-10-2022, and a description of "Seedless grapes".
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -57,26 +56,29 @@ This User Guide covers:
...
+
 Then, add a recipe that uses that ingredient:
 
 `add recipe -n Grape juice -i Grapes 50g -s Mash grapes -s Strain juice`
 
+This adds a recipe named Grape juice, which uses 50g of grapes, with two steps of preparation.
```
Comment (NayLin-H99) [inline docs/UserGuide.md:69]:
```suggestion
This adds a recipe named "Grape juice", which uses 50g of grapes, with two steps of preparation.
```

File: docs/UserGuide.md (hunk 1 above)
Comment (NayLin-H99) [reply docs/UserGuide.md:21]: I think it should either be "Recipe Book" or "recipes" here. 🐒

**Review by NayLin-H99 — COMMENTED** [review]
LGTM apart from a few minor suggestions.

**Review by pattan9955 — APPROVED** [review]
LGTM! 🐒

**Pull request merged**